cmd/tests.go

```diff
@@ -1,95 +1,19 @@
 package cmd
 
 import (
-	"fmt"
-
 	"github.com/CircleCI-Public/circleci-cli/proxy"
-	"github.com/CircleCI-Public/circleci-cli/settings"
-	"github.com/bmatcuk/doublestar"
 	"github.com/spf13/cobra"
 )
 
-type testsOptions struct {
-	cfg  *settings.Config
-	args []string
-}
-
-func newTestsCommand(config *settings.Config) *cobra.Command {
-	opts := testsOptions{
-		cfg: config,
-	}
-
+func newTestsCommand() *cobra.Command {
 	testsCmd := &cobra.Command{
-		Use:    "tests",
-		Short:  "Collect and split files with tests",
-		Hidden: true,
-	}
-
-	globCmd := &cobra.Command{
-		Use:   "glob",
-		Short: "Glob files using pattern",
-		PreRun: func(cmd *cobra.Command, args []string) {
-			opts.args = args
-		},
-		RunE: func(_ *cobra.Command, _ []string) error {
-			return globRun(opts)
+		Use:                "tests",
+		Short:              "Collect and split tests so they may be run in parallel.",
+		DisableFlagParsing: true,
```

Comment:
This flag ensures that when `circleci tests --help` is called, the `--help` flag is passed to the agent instead of being handled by the CLI. That way the help text includes the subcommands.